JobConfig/beam/mustop.fcl

```diff
@@ -0,0 +1,92 @@
+// Generate all muon stop emission products
+//
+// Andy Edmonds, 2020
+
+#include "JobConfig/beam/prolog.fcl"
+
+BEGIN_PROLOG
+
+dioGenTool : {
+  tool_type : DIOGenerator
+  physics : {
+    spectrumShape : Czarnecki
+    spectrumVariable : totalEnergy
+    pdgId : 11 # Czarnecki option in BinnedSpecturm needs pdgId here
+    elow : 1. //MeV
+    spectrumResolution : 0.1 //MeV
+  }
+}
+
+muonCaptureProtonGenTool : { 
+  tool_type : MuCapProtonGenerator
+  physics : {
+    spectrumShape : ejectedProtons 
+    spectrumVariable : kineticEnergy
+    nbins: 1000
+  }
+}
+
+muonCaptureDeuteronGenTool : { 
+  tool_type : MuCapDeuteronGenerator
+  physics : {
+    spectrumShape : ejectedProtons 
+    spectrumVariable : kineticEnergy
+    nbins: 1000
+  }
+}
+
+muonCaptureNeutronGenTool : {
+  tool_type : MuCapNeutronGenerator
+  physics: {
+    spectrumShape: tabulated
+    spectrumFileName: "ConditionsService/data/neutronSpectrum.txt"
+    spectrumVariable: kineticEnergy
+  }
+}
+
+muonCapturePhotonGenTool : {
+  tool_type : MuCapPhotonGenerator
+  physics : {
+    spectrumShape: flat
+    spectrumVariable: totalEnergy
+    elow: 0.0 // MeV
+    ehi: 7.0  // MeV
+  }
+}
+
+# An example that isn't used in this fcl
+#muonCapture1809keVGammaGenTool : {
+#  tool_type : MuCap1809keVGammaGenerator
+#}    
+
+END_PROLOG
+
+process_name: mustop
+
+source: { module_type : EmptyEvent }
+services: { @table::Services.Sim }
+physics: { @table::mu2e.physics.g4s4Muons }
+outputs: { @table::mu2e.outputs.g4s4Defs }
+
+physics.producers.generate: {
+   module_type: MuStopProductsGun
+   verbosityLevel : 1
+   stops : @local::mu2e.tgtMuonStops
+   captureProducts : [ @local::muonCaptureProtonGenTool,
+   		       @local::muonCaptureDeuteronGenTool,
+		       @local::muonCaptureNeutronGenTool,
+		       @local::muonCapturePhotonGenTool
+   		     ]
+   decayProducts : [ @local::dioGenTool ] 
+}
+
+services.TFileService.fileName: "nts.owner.mustop.version.sequencer.root"
+outputs.detectorOutput.fileName: "sim.owner.mustop-TrkCal.version.sequencer.art"
+outputs.crvOutput.fileName: "sim.owner.mustop-CRV.version.sequencer.art"
+outputs.truncatedEvtsOutput.fileName: "sim.owner.mustop-truncated.version.sequencer.art"
+
+physics.producers.generate.stops.inputFiles : [ "mergedMuonStops/nts.mu2e.DS-TGTstops.MDC2018a.001002_00000000.root" ]
```

Comment:
That's another hardcoded input file.  I think production job configs should have @nil for that.

Reply:
Done in commit 17349b7c2